### PR TITLE
reorder readme and add one-line installation command

### PR DESCRIPTION
**Pull request by egpbos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
+# DIANNA: Deep Insight And Neural Network Analysis
+
+## How to use dianna
+
+The project setup is documented in [project_setup.md](project_setup.md). Feel free to remove this document (and/or the link to this document) if you don't need it.
+
+## Installation
+
+To install dianna directly from GitHub repository, do:
```
Comment (loostrum) [inline README.md:9]: There's a missing "the" here:  "from the GitHub repository"

**Review by loostrum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (egpbos) [reply README.md:9]:
```suggestion
To install dianna directly from the GitHub repository, do:
```

Comment (egpbos) [reply README.md:9]: Can you commit that suggestion? I can't :D Otherwise, maybe if you make the suggestion I can merge it...

Comment (loostrum) [reply README.md]: Oh I didn't know that was a feature! Committed and I'll merge it

**Pull request merged**